public-api/src/main/java/org/pmiops/workbench/cdr/CdrDbConfig.java

```diff
@@ -59,6 +59,7 @@ public CdrDataSource(CdrVersionDao cdrVersionDao,
       Map<Object, Object> cdrVersionDataSourceMap = new HashMap<>();
       for (DbCdrVersion cdrVersion : cdrVersionDao.findAll()) {
         int slashIndex = originalDbUrl.lastIndexOf('/');
+        // String dbUrl = "jdbc:mysql:///databrowsermaindb?cloudSqlInstance=aou-db-test:us-central1:databrowsermaindb&socketFactory=com.google.cloud.sql.mysql.SocketFactory";
         String dbUrl =
```

Comment:
You need to put CDR DB name inside 
```
jdbc:mysql:///databrowsermaindb?cloudSqlInstance=aou-db-test:us-central1:{db name}&socketFactory=com.google.cloud.sql.mysql.SocketFactory
```
You need to make sure you are not dropping the part after DB, like `&socketFactory=com.google.cloud.sql.mysql.SocketFactory`

Reply:
Can i just do this for now. Comment out the original part. Put in the test url and test that out.